tools/dm/query-status.md

```diff
@@ -109,35 +109,6 @@ This document introduces the query result and subtask status of Data Migration (
                 "result": null
             }
         },
-        {
```

Comment:
I see that this section is in [docs-cn](https://pingcap.com/docs/tools/dm/query-status/). Are you sure that we need to remove it?

Reply:
This section appeared twice, so we need to remove one occurence.

Reply:
Got it!

Reply:
Got it!